public/site.webmanifest

```diff
@@ -48,8 +48,38 @@
             "type": "image/png"
         }
     ],
+    "shortcuts": [
+        {
+            "name": "Brew AeroPress",
+            "short_name": "AeroPress",
+            "description": "Brew a cup of coffee using the AeroPress",
+            "url": "/aeropress",
+            "icons": [
+                { "src": "/assets/icons/aeropress.png", "sizes": "256x256" }
+            ]
+        },
+        {
+            "name": "Brew French Press",
```

Comment:
Let's remove the "Brew" from here and make the longer name equal to the shorter name.